bentoml/_internal/bento/bento.py

```diff
@@ -30,6 +30,7 @@
 from .build_config import BentoBuildConfig
 from ..configuration import BENTOML_VERSION
 from ..configuration.containers import BentoMLContainer
+from ..runner.resource import Resource
```

Comment:
should we do this under `TYPE_CHECKING` and `from __future__ import annotations`?